src/outcomeFactory.js

```diff
@@ -0,0 +1,41 @@
+const FinalOutcome = require('./finalOutcome')
+const ActionableOutcome = require('./actionableOutcome')
+const NullOutcome = require('./nullOutcome')
+const MarkerFinder = require('./markerFinder')
+
+module.exports = class OutcomeFactory {
```

Comment:
I don't think you're going to be able to get away from including this in both the GameScenario and the top level game. Both places need a way to determine if there's a winner and you definitely want to reuse the same logic for both.